### PR TITLE
Change deprecated sonar.scoverage.reportPath property

### PR DESCRIPTION
**Pull request by ChocPanda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,8 +43,8 @@ You can define your project properties either in the external config file `sonar
  - **sonar.sources** - default Scala source directory relative to the root of your project (usually `src/main/scala`, uses the value of `scalaSource in Compile`)
  - **sonar.tests** - default Scala tests directory relative to the root of your project (usually `src/test/scala`, uses the value of `scalaSource in Test`)
  - **sonar.scala.version** - defines the version of Scala used in your project (i.e. `scalaVersion`)
- - **sonar.scoverage.reportPath** - relative path to the scoverage report (e.g. `target/scala-2.12/scoverage-report/scoverage.xml`)
- - **sonar.scala.scapegoat.reportPath** - relative path to the scapegoat report (e.g. `target/scala-2.12/scoverage-report/scoverage.xml`)
+ - **sonar.scala.coverage.reportPath** - relative path to the scoverage report (e.g. `target/scala-2.12/scoverage-report/scoverage.xml`)
+ - **sonar.scala.scapegoat.reportPath** - relative path to the scapegoat report (e.g. `target/scala-2.12/scapegoat-report/scapegoat.xml`)
```
Comment (mwz) [inline README.md]: Well spotted, thanks!

File: README.md [hunk 2]
```diff
@@ -43,8 +43,8 @@ You can define your project properties either in the external config file `sonar
  - **sonar.sources** - default Scala source directory relative to the root of your project (usually `src/main/scala`, uses the value of `scalaSource in Compile`)
  - **sonar.tests** - default Scala tests directory relative to the root of your project (usually `src/test/scala`, uses the value of `scalaSource in Test`)
  - **sonar.scala.version** - defines the version of Scala used in your project (i.e. `scalaVersion`)
- - **sonar.scoverage.reportPath** - relative path to the scoverage report (e.g. `target/scala-2.12/scoverage-report/scoverage.xml`)
- - **sonar.scala.scapegoat.reportPath** - relative path to the scapegoat report (e.g. `target/scala-2.12/scoverage-report/scoverage.xml`)
+ - **sonar.scala.coverage.reportPath** - relative path to the scoverage report (e.g. `target/scala-2.12/scoverage-report/scoverage.xml`)
```
[the remark this answers is not in the document]
Comment (ChocPanda) [reply README.md]: I've given it a go, but I've never written an sbt-plugin so I wasn't sure about the scripted tests so I just followed the advice in the issue you raised.

[the remark this answers is not in the document]
Comment (BalmungSan) [reply README.md]: Shouldn't it be `sonar.scala.scoverage.reportPath` instead? (_**s**_coverage_ vs _coverage_).

[the remark this answers is not in the document]
Comment (mwz) [reply README.md]: oh yes, that's true - it should be `sonar.scala.scoverage.reportPath`
see https://github.com/mwz/sonar-scala/blob/master/src/main/scala/com/mwz/sonar/scala/scoverage/ScoverageSensor.scala#L164 for a reference

**Comment by ChocPanda** [comment]
Thanks for your help on this

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ChocPanda) [reply README.md]: Spotted and I believe this has been resolved

**Comment by ChocPanda** [comment]
@mwz when do you think you'll have time to merge this?

**Comment by mwz** [comment]
Thanks for making the change @ChocPanda. I'll make the release later today.

**Comment by mwz** [comment]
I've released this in `1.7.0`, thanks again for your contribution @ChocPanda.

**Pull request merged**